AsyncDisplayKit/ASVideoNode.mm

```diff
@@ -126,14 +126,17 @@ - (AVPlayerItem *)constructPlayerItem
 {
   ASDN::MutexLocker l(__instanceLock__);
 
-  if (_asset != nil) {
-    AVPlayerItem *playerItem = [[AVPlayerItem alloc] initWithAsset:_asset];
-    playerItem.videoComposition = _videoComposition;
-    playerItem.audioMix = _audioMix;
-    return playerItem;
+  AVPlayerItem *playerItem = nil;
+  if (self.assetURL != nil) {
```

Comment:
@Xinchi I'm not sure this is correct - what if we already have an _asset, which was created from this assetURL?